doc/configuration-v2_1-experimental.md

```diff
@@ -39,9 +39,11 @@ The Ignition configuration is a JSON document conforming to the following specif
     * **_mount_** (object): contains the set of mount and formatting options for the filesystem. A non-null entry indicates that the filesystem should be mounted before it is used by Ignition.
       * **device** (string): the absolute path to the device. Devices are typically referenced by the `/dev/disk/by-*` symlinks.
       * **format** (string): the filesystem format (ext4, btrfs, xfs, or swap).
-      * **_create_** (object): contains the set of options to be used when creating the filesystem. A non-null entry indicates that the filesystem shall be created.
-        * **_force_** (boolean): whether or not the create operation shall overwrite an existing filesystem.
-        * **_options_** (list of strings): any additional options to be passed to the format-specific mkfs utility.
+      * **_force_** (boolean): whether or not the create operation shall overwrite an existing filesystem.
```

Comment:
This like was the `force` option being moved out of the `create` object. Should it not have the same semantics?